docs/examples/grpc/README.md

```diff
@@ -1,93 +1,141 @@
+  ```
+  $ kubectl get po -A -o wide | grep go-grpc-greeter-server
+  ```
+- If you have a gRPC app deployed in your cluster, then skip further notes in this Step 1, and continue from Step 2 below. 
+
+- As an example grpc application, we can use this official gRPC app <https://github.com/grpc/grpc-go/blob/91e0aeb192456225adf27966d04ada4cf8599915/examples/features/reflection/server/main.go> .
+
+- To create a container image for this app, you can use [this Dockerfile](../../../images/go-grpc-greeter-server/rootfs/Dockerfile). 
+
+- Currently, an image is built using this Dockerfile and published on hub.docker.com as `longwuyuan/go-grpc-greeter-server`. Soon, this will change, and then this document will be updated accordingly.
+
+- If you use the Dockerfile mentioned above, to create a image, then given below is an example of a Kubernetes manifest, to create a deployment resource, that uses that image. If needed, then edit this manifest to suit your needs. Assuming the name of this yaml file is  `deployment.go-grpc-greeter-server.yaml` ;
+
+  ```
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    labels:
+      app: go-grpc-greeter-server
+    name: go-grpc-greeter-server
+  spec:
+    replicas: 1
+    selector:
+      matchLabels:
+        app: go-grpc-greeter-server
+    template:
+      metadata:
+        labels:
+          app: go-grpc-greeter-server
+      spec:
+        containers:
+        - image: longwuyuan/go-grpc-greeter-server
+          name: go-grpc-greeter-server
+          ports:
+          - containerPort: 50051
+  ```
+
+### Step 2: Create the Kubernetes `Service` for the gRPC app
+
+- You can use the following example manifest to create a service of type ClusterIP. Edit the name/namespace/label/port to match your deployment/pod ;
+  ```
+  apiVersion: v1
+  kind: Service
+  metadata:
+    labels:
+      app: go-grpc-greeter-server
+    name: go-grpc-greeter-server
+  spec:
+    ports:
+    - port: 80
+      protocol: TCP
+      targetPort: 50051
+    selector:
+      app: go-grpc-greeter-server
+    type: ClusterIP
+  ```
+- You can save the above example manifest to a file with name `service.go-grpc-greeter-server.yaml` and edit it to match your deployment/pod, if required. You can create the service resource with a kubectl command like this ;
+
+  ```
+  $ kubectl create -f service.go-grpc-greeter-server.yaml
+  ```
+
+### Step 3: Create the Kubernetes `Ingress` resource for the gRPC app
+
+- Use the folowing example manifest for a ingress resource to create a ingress for your grpc app. Edit it if required to match your app's details like name, namespace, servicename, tls-secretName etc.. Make sure you have the required SSL-Certificate, existing in your Kubernetes cluster's namespace, as a secret resource/object of type tls. This is because we are terminating TLS on the ingress, in this example ;
+
+  ```
+  apiVersion: networking.k8s.io/v1beta1
```

Comment:
we are moving the ingress to v1, so we should already point here to networking.k8s.io/v1 instead of v1beta1

Reply:
ok that was stupid of me. I used v1 syntax in rule and did not change apiVersion. Now fixed.